lib/parsers/open-api.js

```diff
@@ -126,6 +120,26 @@ class JoiOpenApiSchemaParser extends JoiJsonSchemaParser {
       delete fieldSchema.type
     }
   }
+
+  _copyMetasToSchema(joiSpec, schema) {
+    if (!_.isEmpty(joiSpec.metas)) {
+      _.each(joiSpec.metas, meta => {
+        _.each(meta, (value, key) => {
+          if (key.startsWith('x-') || key === 'deprecated' || key === 'readOnly' || key === 'writeOnly') {
+            schema[key] = value
+          }
+        })
+      })
+    }
+  }
```

Comment:
Do we want to do this for any other of the targets?

Reply:
Other target should be appropriate as well, thanks.

Reply:
Done in 1f8f068